cli/types/cypress.d.ts

```diff
@@ -10,10 +10,12 @@ declare namespace Cypress {
   type PrevSubject = keyof PrevSubjectMap
   type TestingType = 'e2e' | 'component'
   type PluginConfig = (on: PluginEvents, config: PluginConfigOptions) => void | ConfigOptions | Promise<ConfigOptions>
-
+	interface JQueryWithSelector<TElement = HTMLElement> extends JQuery<TElement> {
```

Comment:
@atarek12 Mind taking a pass through your changes and correcting the whitespace differencing that you introduced?